src/data/padics/padic_numbers.lean

```diff
@@ -449,21 +485,21 @@ cau_seq.completion.of_rat_div
 
 @[simp] lemma of_rat_zero : of_rat p 0 = 0 := rfl
 
-@[simp] lemma cast_eq_of_rat_of_nat (n : ℕ) : (↑n : ℚ_[p]) = of_rat p n :=
+lemma cast_eq_of_rat_of_nat (n : ℕ) : (↑n : ℚ_[p]) = of_rat p n :=
 begin
   induction n with n ih,
   { refl },
   { simpa using ih }
 end
 
-@[simp] lemma cast_eq_of_rat_of_int (n : ℤ) : ↑n = of_rat p n :=
```

Comment:
Is there a module docstring documentation of the choices of simp normal form here?

Reply:
Despite writing this file I can't say I'm an expert, hah. I added a short note. I have no idea why this was a simp lemma, it's very annoying.